crates/sui-core/src/checkpoints/mod.rs

```diff
@@ -301,6 +306,82 @@ impl CheckpointStore {
         self.full_checkpoint_content.get(&seq)
     }
 
+    fn prune_local_summaries(&self) -> SuiResult {
+        if let Some((last_local_summary, _)) = self
+            .locally_computed_checkpoints
+            .unbounded_iter()
+            .skip_to_last()
+            .next()
+        {
+            let mut batch = self.locally_computed_checkpoints.batch();
+            batch.delete_range(&self.locally_computed_checkpoints, &0, &last_local_summary)?;
+            batch.write()?;
+            info!("Pruned local summaries up to {:?}", last_local_summary);
+        }
+        Ok(())
```

Comment:
Could this be just `self.locally_computed_checkpoints.clear()` ?

Reply:
yes, good call. not sure why i made it so difficult lol